lms/resources/_js_config/__init__.py

```diff
@@ -436,7 +436,10 @@ def _hypothesis_client(self):
         }
 
     def _groups(self):
-        if self._context.canvas_sections_enabled or self._context.canvas_groups_enabled:
+        if (
+            self._context.canvas_sections_enabled
+            or self._context.canvas_is_group_launch
+        ):
```

Comment:
There's a very similar if statement in `_sync_api`, I'm just pattern matching here, but does that need to change too?